.github/workflows/slack-notify.yml

```diff
@@ -0,0 +1,39 @@
+on:   
+  discussion:
+    types: [created]  
+name: Slack Notification Demo
```

Comment:
One last change and I will approve this PR.

Could we call it

```
name: Github discussion Notification
```

or something like this

Reply:
Did you see this change @Gordon-BP?

Reply:
Yes that change is OK